ocaml/quicktest/quicktest_max_vdi_size.ml

```diff
@@ -0,0 +1,46 @@
+
+(** From XenServer 7.5 Configuration Limits
+    https://docs.citrix.com/content/dam/docs/en-us/xenserver/current-release/downloads/xenserver-config-limits.pdf *)
+let max_vdi_size = Sizes.((2L ** tb) -* (4L ** gb))
```

Comment:
Is it `TB` or `TiB`? Not sure which one the config limits meant, since this is a limit due to sizes of integers I would assume that it would somehow be related to powers of 2, and thus `tib` and `gib` instead of `tb` and `gb`.
We should check which one works, and update the config limits docs to be correct.

Reply:
I just assumed that the document is correct, and it's `TB`.
But I'll check, so that we know what is the truth. I wouldn't change the document based on one experiment, I think more testing goes into verifying these limits. I just viewed this document as a contract of what XenServer should be able to do. So I'd rather not change it now, because probably other people have also been relying on this contract.

But if it supports `TiB`, I can maybe add another "stress test" with this limit.

Reply:
It failed:
```
[ERROR]             Quicktest_max_vdi_size                    0   VDI.copy on SR [NFS Storage Repository 1].
[ERROR]             Quicktest_max_vdi_size                    1   VDI.export/import to/from VHD file on SR [NFS Storage Repository 1].
[OK]                Quicktest_static_vdis                     0   test_static_vdis for SMAPIv1 on SR [NFS Storage Repository 1].
-- Quicktest_max_vdi_size.000 [VDI.copy on SR [NFS Storage Repository 1].] Failed --
in _build/_tests/Quicktest_max_vdi_size.000.output:
[exception] Server_error(SR_BACKEND_FAILURE_79, [ ; VDI Invalid size [opterr=VDI size must be between 1 MB and 2093050 MB];  ])
Raised at file "ocaml/xapi-client/client.ml", line 6, characters 31-75
Called from file "ocaml/xapi-client/client.ml", line 18, characters 55-110
Called from file "ocaml/xapi-client/client.ml", line 11506, characters 6-55
Called from file "ocaml/quicktest/qt.ml", line 125, characters 4-321
Called from file "ocaml/quicktest/qt.ml", line 146, characters 15-51
Called from file "src/alcotest.ml", line 269, characters 8-14


The full test results are available in `_build/_tests`.
2 errors! in 59.932s. 92 tests run.
Fatal error: exception Alcotest.Test_error
Raised at file "lib/xapi-stdext-pervasives/pervasiveext.ml", line 26, characters 6-15
Called from file "ocaml/quicktest/quicktest.ml", line 20, characters 2-1023
```

Reply:
So I'd leave the test as it is, because `TiB` doesn't work

Reply:
Ok good to know, can you add a comment saying that TiB doesn't work and TB is the correct limit?

Reply:
Sure, done.